data/coalition outfits.txt

```diff
@@ -204,9 +204,9 @@ outfit "Large Steering Module"
 	"turn" 711.9
 	"turning energy" 1.0
 	"turning heat" 8.2
-	description "By focusing on small, modular engines rather than the massive capital ship drives that most other species develop, the Coalition loses some of the potential efficiency for very large ships, but gains a considerable amount of flexibility."
+	description "By focusing on small, modular engines rather than the massive capital ship drives that most other species develop, the Coalition loses some of the potential efficiency for very large ships but gains a considerable amount of flexibility. Some captains are fine with installing multiple, but some want a new model to be released."
 
 outfit "Coalition License"
 	category "Special"
 	thumbnail "outfit/coalition license"
-	description "Completed a lot of Coalition jobs and received permission to purchase civilian technology and ships."
+	description "Completed a lot of Coalition jobs and received permission to purchase civilian technology and ships except for their weapons, reactor, and warships."
```

Comment:
This isn't necessary.
`received permission to purchase civilian technology` already explicitly excludes `their weapons, reactor[s], and warships`.